### ISSUE TITLE
Allow logos to be stored for each object

### ISSUE THREAD

**Issue by DeleMike** [MLH-Fellowship/orientation-project-python-23.SUM.B.1#7]
All Experience, Education and Skill objects contain a logo attribute. Figure out a way to store these images with user friendliness in mind.

The client should identify if the logo field is empty and display this field.

**Comment by DeleMike** [comment]
@marcusgchan, I was thinking... what if from the frontend, when the user adds or edit any logo, we save this logo by uploading the picture to firebase storage or cloudinary free storage service then once the upload operation is done and successful, we can get an image url which we can use to store in our database.

When the frontend wants to display a logo, it can use this image url that is our database.

**Comment by marcusgchan** [comment]
Ok, are you thinking about using presigned urls for upload or sending the image to the backend and then uploading it?

**Comment by DeleMike** [comment]
Yes, presigned url if it means that the front end saves the image (Logo) to some storage and then we can get uses the resigned URL to add an experience, skill or education. this is okay?
